keystrokes/src/components/Game/App.js

```diff
@@ -1,14 +1,14 @@
 import React, { Component, Fragment } from 'react';
-import './App.css';
+import '../../stylesheets/App.css';
 
-import { getRandomKey } from './utils/utils.js';
+import { getNextKey } from '../utils/utils.js';
 
-import { ref } from './config/constants';
+import { ref } from '../../config/constants';
 
 class App extends Component {
 	constructor() {
 		super();
-		this.getRandomKey = getRandomKey.bind();
+		this.getNextKey = getNextKey.bind();
```

Comment:
This is fine, but i saw you using arrow functions to bind somewhere else. I'd change it personally.

Reply:
I think i had issues making this work in here. Would you just define it the same as handle key press?

Reply:
From App